src/scene/index.js

```diff
@@ -73,7 +73,7 @@ export function isHovering() {
 export function render() {
   const freeze = !hover || drag || !GLOBE_HOVER_FREEZE_ENABLED;
   if (freeze) {
-    target.x += 0.00075;
+    target.x += 0.001185;
```

Comment:
We could make this a constant so it's more easily configurable (GLOBE_ROTATION_SPEED or something) 🐒

Reply:
Done 👍